### PR TITLE
Use `email_address_with_name`

### PR DESCRIPTION
**Pull request by andyw8**
https://boringrails.com/tips/rails-mailers-email-address-with-name

Hat tip to @swanson

### CODE REVIEW

**Review by pirj — APPROVED** [review]
❤️

File: README.adoc [hunk 1]
```diff
@@ -1425,12 +1425,11 @@ default_url_options[:host] = 'your_site.com'
...
 
 Format the from and to addresses properly.
-Use the following format:
 
 [source,ruby]
 ----
 # in your mailer class
-default from: 'Your Name <info@your_site.com>'
+default from: email_address_with_name('Your Name', 'info@your_site.com')
 ----
```
Comment (koic) [inline README.adoc:1433]: Good catch! OTOH, I think it's better not to remove it for Rails 6.0 (or lower) users. How about this?

```diff
 Format the from and to addresses properly.
 Use the following format:

 [source,ruby]
 ----
 # in your mailer class
 default from: 'Your Name <info@your_site.com>'
 ----

+If you're using Rails 6.1 or higher, you can use `email_address_with_name` method.
+
+[source,ruby]
+----
+default from: email_address_with_name('Your Name', 'info@your_site.com')
+----
```

Comment (andyw8) [reply README.adoc:1433]: @koic thanks, updated.

**Comment by koic** [comment]
Thanks!

**Pull request merged**